File: LTHAM/061/061.cpp
#include <iostream>
using namespace std;

int SoLuongChuSoLe(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;
	
	cout << "So luong chu so le la: " << SoLuongChuSoLe(n) << endl;
	return 0;
}

int SoLuongChuSoLe(int nn)
{
	int dem = 0;
	int t = nn;
	while (t != 0)
	{
		int dv = t % 10;
		if (dv % 2 == 1)
			dem++;
		t = t / 10;
	}
	return dem;
}

File: CTDK_Python/061/_061.py
n = int(input("Nhap n: "))

dem = 0;
t = n
while(t != 0):
    dv = t % 10
    if dv % 2 != 0:
        dem = dem + 1
    t = t // 10

print("So luong cac chu so le: ", s)
